src/ovp_pipeline/truth_api.py

```
@@ -819,17 +819,20 @@ def get_object_provenance_map(
         mention_rows = conn.execute(
             f"""
-            SELECT page_links.target_slug, pages_index.slug, pages_index.title, pages_index.note_type, pages_index.path
+            SELECT page_links.target_slug, pages_index.slug, pages_index.title, pages_index.note_type, pages_index.path,
+                   objects.object_kind
             FROM page_links
             JOIN pages_index ON pages_index.slug = page_links.source_slug
+            LEFT JOIN objects ON objects.object_id = pages_index.slug AND objects.pack = ?
             WHERE page_links.target_slug IN ({placeholders})
               AND pages_index.slug != page_links.target_slug
             ORDER BY page_links.target_slug, pages_index.slug
             """,
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)